conf/defaults.yaml

```diff
@@ -49,6 +49,9 @@ storm.nimbus.retry.times: 5
 storm.nimbus.retry.interval.millis: 2000
 storm.nimbus.retry.intervalceiling.millis: 60000
 storm.auth.simple-white-list.users: []
+storm.auth.simple-acl.users: []
```

Comment:
thanks.. i think it that got introduced somehow during manual reconciling/rebasing to latest master.